README.md

```diff
@@ -29,6 +29,8 @@ standard OS temporary directory, then you are free to override that as well.
 
 ## An Important Note on Compatibility
 
+See the [CHANGELOG](./CHANGELOG.md) for more information.
```

Comment:
We should try to automate it with [github-changelog](https://github.com/whitepages/github-changelog).

Reply:
maybe you can also publish it to github releases
https://github.com/raszi/node-tmp/releases

Reply:
let us take this step by step, please.
the current code base needs a lot of attention in order to stabilize it.
but I am game for that :)

Reply:
as for github-changelog, it uses conventional-changelog, which I am trying to implement with the next release, if there be any.

Reply:
and automation of the changelog cannot be done right now as the commit messages do not match the existing patterns.

Reply:
as for releases, this I cannot do as I have limited access to this repo, only.

Reply:
> and automation of the changelog cannot be done right now as the commit messages do not match the existing patterns.

The linked project is using the pull-request titles not the commit messages, so it can be adapted. I was trying to make sure that all of the PR titles are in that format but those can be modified anytime.